### ISSUE TITLE
Deprecate this engine

### ISSUE THREAD

**Issue by kprinssu** [blankmac/VoodooI2CUPDDEngine#3]
Considering that VoodooInput is a much better alternative, can we consider deprecating this?

**Comment by blankmac** [comment]
Agreed.  I can't imagine why someone would use this vs VoodooInput.
